POGOLib.Official/Net/RpcEncryption.cs

```
@@ -190,8 +190,11 @@ private List<LocationFix> BuildLocationFixes(RequestEnvelope requestEnvelope, lo
             var requestsBytes = requestEnvelope.Requests.Select(x => x.ToByteArray()).ToArray();
             var hashData = await Configuration.Hasher.GetHashDataAsync(requestEnvelope, signature, locationBytes, requestsBytes, serializedTicket);
 
-            signature.LocationHash1 = (int) hashData.LocationAuthHash;
-            signature.LocationHash2 = (int) hashData.LocationHash;
+            if (hashData == null)
+                return null;
```

Maybe better to throw an exception because `null` is not expected here.

---

Aff this please :-)

---

Add*